ldm/invoke/model_cache.py

```diff
@@ -125,78 +128,75 @@ def list_models(self) -> dict:
                        },
           model_name2: { etc }
         '''
-        result = {}
         for name in self.config:
             try:
                 description = self.config[name].description
             except ConfigAttributeError:
                 description = '<no description>'
+
             if self.current_model == name:
                 status = 'active'
             elif name in self.models:
                 status = 'cached'
             else:
                 status = 'not loaded'
-            result[name]={}
-            result[name]['status']=status
-            result[name]['description']=description
-        return result
+
+            return {
+                name: {
+                    'status': status,
+                    'description': description,
+                }}
```

Comment:
This is incorrect. list_models() is supposed to return a list of all the configured models, not the last one.